aim/cli/manager/manager.py

```diff
@@ -45,7 +45,7 @@ def run_up(args):
         'host': 'http://' + args['--host']
     }
 
-    time.sleep(2)
+    time.sleep(5)
```

Comment:
how stable is this? 🤔
is there a robust way to make sure the aim up is running?

Reply:
Changed to `/status` api checks instead, I guess this one is more trustworthy